feedsx/src/main/java/com/likeminds/feedsx/utils/customview/ViewDataBinders.kt

```diff
@@ -15,7 +14,7 @@ abstract class ViewDataBinder<V : ViewDataBinding, T : BaseViewType> {
 
     abstract fun bindData(binding: V, data: T, position: Int)
 
-    fun bindData(binding: V, data: Bundle, position: Int) {
+    fun bindData() {
```

Comment:
undo this change, we will use this `bindData()` once we add `DiffUtils` in recycler view to update the data